Muhire-Josue/freelancerz
# feature(profile)

#### What does this PR do?

- It creates a profile for the developer

#### Description of Task to be completed?

- When the developer signup a profile is created based on the GitHub and the provided data

#### How should this be manually tested?

- Clone: `git clone https://github.com/Muhire-Josue/freelancerz.git`

- Install dependencies: `npm install`

- Check into this branch: `ft-developer-profile-#173764078`

- Run the server: `npm run dev` ==> should get this message in the terminal `Listening on port 3000`

- Hit this URL in postman to disable email notifications `PUT: http://localhost:3000/api/profile`

- Send this data
```
{
     "userId": 1
}
```

#### Any background context you want to provide?

- make sure you have [node v10 - v13](https://nodejs.org/en/download/) not node v14

- make sure you have [Postgres](https://www.postgresql.org/download/) up and running

- create .env file and use the .env.example file as a reference to create the variable environment needed for this project
#### What are the relevant pivotal tracker stories?

- [173764078](https://www.pivotaltracker.com/story/show/173764078)

#### Screenshots (if appropriate)
![profile](https://user-images.githubusercontent.com/37122177/88893498-bbe34980-d246-11ea-8866-599a24deef8f.PNG)
